### PR TITLE
[Refactor] Updated Dashboard page

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: Yash Kamboj <yashkamboj29@gmail.com>

**Description**Updated Dashboard page ,
1. removed release and version section
2. added install service mesh section 
3. changed metrics button flex-direction
Screenshot (129)

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/pages/index.js [hunk 1]
```diff
@@ -67,66 +68,49 @@ export default function Dashboard() {
...
+  >
+    <Typography style={{ fontSize : "1.5rem", marginBottom : "2rem" }} align="center" color="textSecondary">
+              No service meshes detected in the cluster.
+              </Typography>
+              <Button
+                aria-label="Add Meshes"
+                variant="contained"
+                color="primary"
+                size="large"
+                // onClick={() => self.props.router.push("/management")}
```
Comment (adithyaakrishna) [inline ui/pages/index.js:99]: @YashKamboj Can we remove this line?

Comment (YashKamboj) [reply ui/pages/index.js]: @adithyaakrishna Done!!!

**Comment by adithyaakrishna** [comment]
@YashKamboj Let's wait for @sudo-NithishKarthik or @Chadha93 's review on this and then will Merge it

File: ui/pages/index.js [hunk 2]
```diff
@@ -67,66 +68,48 @@ export default function Dashboard() {
...
+    <div
+    style={{
+      margin: "auto",
+      display : "flex",
+      justifyContent : "center",
+      alignItems : "center",
+      flexDirection : "column",
+    }}
+  >
+    <Typography style={{ fontSize : "1.5rem", marginBottom : "2rem" }} align="center" color="textSecondary">
```
Comment (Chadha93) [inline ui/pages/index.js:91]: @YashKamboj Could we avoid inline styles in refactoring? I know we are using [styled API](https://mui.com/system/styled/) so no` makestyles `will be used. We could probably use this `sx` prop. Reference - https://mui.com/system/the-sx-prop/#typography or use plain CSS separate sheet ref- https://mui.com/material-ui/migration/migration-v4/#fix-css-specificity-optional.

Let's define all the criteria to get the best out of this refactoring process.

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**